studio/app/common/routers/workflow.py

```diff
@@ -87,3 +88,28 @@ async def import_workflow_config(file: UploadFile = File(...)):
         return WorkflowConfig(**contents)
     except Exception as e:
         raise HTTPException(status_code=400, detail=f"Parsing yaml failed: {str(e)}")
+
+
+@router.get(
+    "/sample_data",
+    dependencies=[Depends(is_workspace_available)],
+)
+async def copy_sample_data(workspace_id: str):
+    all_files_copied = True  # Initialize flag
+    folders = ["input", "output"]
+    for folder in folders:
+        sample_data_dir = os.path.join("sample_data", folder)
+        if folder == "input":
+            user_dir = os.path.join(DIRPATH.INPUT_DIR, workspace_id)
+        elif folder == "output":
+            user_dir = os.path.join(DIRPATH.OUTPUT_DIR, workspace_id)
+        if not os.path.exists(user_dir):
+            os.makedirs(user_dir)
```

Comment:
we have utility function `create_directory` to do same process.
pls use that.
it can be imported from 
`studio.app.common.core.utils.filepath_creater`